### ISSUE TITLE
JetTabs Dynamic Tab Visibility

### ISSUE THREAD

**Issue by johnnyboland** [Crocoblock/suggestions#931]
Add an option in JetTabs to not display a particularly tab dynamically if the field is empty.  

I think dynamic visibility with tabs is extremely important for making dynamic content tabs viable.

**Comment by Gidsa** [comment]
Yes this would be most helpful. Is there a workaround for the time-being that anyone is aware of?

**Comment by counterspace27** [comment]
+1

**Comment by restre** [comment]
I support this too, it's needed!

**Comment by jarahner** [comment]
+1

**Comment by laportelatine** [comment]
+1
Would be really usefull for me. I already have use case.

**Comment by jefftom** [comment]
Another +1 I thought this feature was already in Crocoblock, why I bought it over toolset.

**Comment by Elsteck** [comment]
+1

**Comment by taidang26** [comment]
+1

**Comment by About2git** [comment]
This is already possible with the "Hide empty items" Addon /Plugin https://crocoblock.com/freemium/tools/

here is the Github: https://github.com/MjHead/jet-hide-empty-items

**Comment by About2git** [comment]
I guess can be closed.

**Comment by limacon12** [comment]
+1

**Comment by limacon12** [comment]
@About2git is right. Here is how to do it:
1) You need JetEngine with enabled "Dynamic visibility" & JetTabs with "Hide empty items" Addon /Plugin https://crocoblock.com/freemium/tools/
2) Your tabs widget must be a JetTabs
3) You create an Elementor template and place dynamic visibility (from JetEngine) logic of the items inside. I do it for the containers. - Example: https://prnt.sc/sIsvYtPOQJ6P
4) Then you attach the template to the desired tab - Example: https://prnt.sc/7M7Zd6RWA6v3

Good luck!

**Comment by aalexandr21** [comment]
Hi

You can use the 'Hide empty items' addon to hide the tab items if the content is empty

Also, you can use the Query Builder to generate the tab items (here an example for Accordion, but it works the same way for tabs https://youtu.be/8fwPAyNwfmw?feature=shared )

**Comment by etasayco** [comment]
Hace tiempo no usaba este plugin y no recordaba que oculta automáticamente los objetos vacíos en los tabs, no se debe hacer nada,  solo instalarlo y probar.
